process/transaction/metaProcessSimulate.go

```diff
@@ -0,0 +1,21 @@
+package transaction
+
+type metaProcessorSimulate struct {
+	*metaTxProcessor
+}
+
+// NewMetaTxProcessorSimulate will create a new instance of meta processor for simulate
+func NewMetaTxProcessorSimulate(args ArgsNewMetaTxProcessor) (*metaProcessorSimulate, error) {
```

Comment:
this is not needed. balanceCheck does not happen on meta tx processor as there are no user account on metachain,

Reply:
removed this.